src/components/core/DpDataTable/DpResizableColumn.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <th
+    :class="`c-data-table__resizable ${isLast? 'u-pr-0' : ''}`"
+    :data-col-idx="idx"
+    v-tooltip="headerField.tooltip || headerField.label">
+    <slot/>
+    <dp-resize-handle
+      v-if="!isLast"
+      :display-icon="isResizableColumn"
+      @mousedown="e => initResize(e, idx)" />
+  </th>
+</template>
+
+<script>
+import DpResizeHandle from './DpResizeHandle'
+import { hasOwnProp } from '../../../utils'
+
+export default {
+  name: 'DpResizableColumn',
+
+  components: {
+    DpResizeHandle
+  },
+
+  props: {
+    idx: {
+      required: true,
+      type: Number
+    },
+
+    headerField: {
+      type: Object,
+      required: true
+    },
+
+    isLast: {
+      type: Boolean,
+      required: false,
+      default: false
+    }
+  },
+
+  data () {
+    return {
+      namedFunc: '',
+      currentHandle: '',
+      nextEl: '',
+      cursorStart: 0,
+      dragStart: false,
+      resize: '',
+      resizeWidth: '',
+      nextWidth: ''
+    }
+  },
+
+  computed: {
+    isResizableColumn () {
+      return hasOwnProp(this.headerField, 'resizeable')? this.headerField.resizeable : true
+    }
+  },
+
+  methods: {
+    initResize (e, idx) {
+      this.resize = document.querySelector(`th[data-col-idx='${idx}']`)
```

Comment:
We can make this more vue-ish and use refs instead

Reply:
done https://github.com/demos-europe/demosplan-ui/pull/205/commits/4c377175718380574178321b328070184a6f8929